### DISCUSSION TITLE
Meldung: Bitte melde dich als Administrator an, um die Registrierung dieser Installation durchzuführen

### DISCUSSION THREAD

**Discussion by Nemo3006** [solectrus/solectrus#2343] [Q&A]
Hallo zusammen, heute habe ich es endlich geschafft, solectrus zu updaten, damit nach dem Senec Update alles wieder gut funktioniert. Nur leider bekomme ich beim Start von Solectrus jetzt die Meldung "Bitte melde dich als Administrator an, um die Registrierung dieser Installation durchzuführen." Kann mir jemand helfen dieses Problem zu beheben? DANKE!

**Comment by ledermann** [comment]
Da gibt es gar kein Problem :)

Zum Admin-Login bitte hier lesen:
https://solectrus.de/faq/admin-zugang/

Zur Registrierung steht hier etwas:
https://solectrus.de/blog/2023-08-13-version-0-13/
